x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/mocks.ts

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProtectionModes } from '../../../../../common/endpoint/types';
+import { GetPolicyListResponse } from '../../policy/types';
+
+export const getPolicyResponse = (): GetPolicyListResponse => ({
+  items: [
+    {
```

Comment:
Do we have a generator for policy response? if so, I would suggest inializing a generator here with a seed and then getting the mocked policy response with it, rather than to define the entire response here.